### PR TITLE
added pushbullet

### PR DESCRIPTION
**Pull request by TimVNL**
Added the Pushbullet App

### CODE REVIEW

File: Casks/pushbullet.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'pushbullet' do
+  version '1.2'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/pushbullet.rb]: `sha256` needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by vitorgalvao** [comment]
This app seems to only be available from the Mac App Store. Where are you getting that link from?

**Comment by alebcay** [comment]
It seems to be unofficial (repackaged). I personally trust the site - the author is well known in the Android community for work on a number of apps, but it's still not the official download source nonetheless.

**Comment by vitorgalvao** [comment]
Seems like we can’t add it, then. Thank you for the information, @alebcay.

@TimVNL Unfortunately, we need official urls to add an app to homebrew-cask (users should at all times be able to verify these links as official, even if us as maintainers trust the alternative sources). Thank you for the contribution, nonetheless.

**Pull request closed without merging**